tasks/section4.yml

```diff
@@ -218,7 +218,10 @@
       - rule_4.1.11
 
 - name: "SCORED | 4.1.12 | PATCH | Ensure use of privileged commands is collected"
-  shell: for i in  $(df | grep '^/dev' | awk '{ print $NF }'); do find $i -xdev -type f -perm -4000 -o -type f -perm -2000 2>/dev/null; done
+  shell: "set -o pipefail && \
```

Comment:
I'd probably use yaml multi-line syntax and skip the quotes.

Reply:
I can change it to use the `>-` multi-line syntax.
The difference is that the newlines in the YAML will be preserved.

```yaml
- name: "..."
  shell: >-
    set -o pipefail &&
    for i in $(df | grep '^/dev' | awk '{ print $NF }'); do
      find $i -xdev -type f -perm -4000 -o -type f -perm -2000 2>/dev/null;
    done
```
yields:

```shell
set -o pipefail && for i in $(df | grep '^/dev' | awk '{ print $NF }'); do
  find $i -xdev -type f -perm -4000 -o -type f -perm -2000 2>/dev/null;
done
```

whereas,
```yaml
- name: "..."
  shell: "set -o pipefail && \
    for i in $(df | grep '^/dev' | awk '{ print $NF }'); do \
      find $i -xdev -type f -perm -4000 -o -type f -perm -2000 2>/dev/null; \
    done"
```
yields:

```shell
set -o pipefail && for i in $(df | grep '^/dev' | awk '{ print $NF }'); do find $i -xdev -type f -perm -4000 -o -type f -perm -2000 2>/dev/null; done
```